data/plugins/drawwhitespace.lua

```diff
@@ -11,6 +11,7 @@ config.plugins.drawwhitespace = common.merge({
   show_leading = true,
   show_trailing = true,
   show_middle = true,
+  show_selected_only = false,
```

Comment:
Not a fan of this name as it suggests it draws *inside* the selection only, while this draws if it's in a line where there is a selection.